src/components/grid-aware/ProgramsPageLargeParagraph/ProgramsPageLargeParagraph.module.css

```
@@ -0,0 +1,67 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: var(--bleed-grid-template-columns);
+  justify-items: center;
+  margin: auto;
+  max-width: var(--bleed-max-width);
+  padding-bottom: 100px;
+  padding-top: 100px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.bleedWrapper > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+    padding-bottom: 40px;
+    padding-top: 30px;
+  }
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  text-align: center;
+}
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: repeat(2, auto);
+}
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
```

Nit: Could you add a blank line between CSS blocks?

```suggestion
}

/* Override grid item default of auto so that item contents cannot stretch the grid itself */
```